examples/quickstart/10-configmaps.jsonnet

```diff
@@ -0,0 +1,233 @@
+      env:
+        - name: NODE_NAME
+          valueFrom:
+          fieldRef:
+            apiVersion: v1
+            fieldPath: spec.nodeName
+        - name: RESULTS_DIR
+          value: /tmp/results
+      image: gcr.io/heptio-images/sonobuoy:latest
+      imagePullPolicy: Always
+      securityContext:
+        privileged: true
+      volumeMounts:
+        - mountPath: /tmp/results
+          name: results
+        - mountPath: /etc/sonobuoy
+          name: config
+  volumes:
+    - name: root
+      hostPath:
+        path: /
+    - name: results
+      emptyDir: {}
+    - name: config
+      configMap:
+        # This will be rewritten when the DaemonSetPlugin driver goes to launch the pod.
+        name: __SONOBUOY_CONFIGMAP__',
+  "e2e.yaml": 'name: e2e
+driver: Job
+resultType: e2e
+spec:
+  serviceAccountName: sonobuoy-serviceaccount
+  tolerations:
+    - key: node-role.kubernetes.io/master
+      operator: Exists
+      effect: NoSchedule
+    - key: CriticalAddonsOnly
+      operator: Exists
+  restartPolicy: Never
+  containers:
+    - name: e2e
+      image: gcr.io/heptio-images/kube-conformance:latest
+      imagePullPolicy: Always
+      # NOTE: Full conformance can take a while depending on your cluster size.
+      # As a result, only a single test is set atm to verify correctness.
+      # Operators that want the complete test results can comment out the
+      # env section.
+      env:
+        - name: E2E_FOCUS
+          value: "Pods should be submitted and removed"
+      volumeMounts:
+        - name: results
+          mountPath: /tmp/results
+    - name: sonobuoy-worker
+      command:
+        - sh
+        - -c
+        - /sonobuoy worker global -v 5 --logtostderr
+      env:
+        - name: NODE_NAME
+          valueFrom:
+          fieldRef:
+              apiVersion: v1
+              fieldPath: spec.nodeName
+        - name: RESULTS_DIR
+          value: /tmp/results
+      image: gcr.io/heptio-images/sonobuoy:latest
+      imagePullPolicy: Always
+      volumeMounts:
+        - name: config
+          mountPath: /etc/sonobuoy
+        - name: results
+          mountPath: /tmp/results
+  volumes:
+    - name: results
+      emptyDir: {}
+    - name: config
+      configMap:
+        # This will be rewritten when the JobPlugin driver goes to launch the pod.
+        name: __SONOBUOY_CONFIGMAP__'
+};
+
+local sonobuoyConfig = configMap.new() +
+    configMap.mixin.metadata.name(conf.sonobuoyCfg.name) +
+    configMap.mixin.metadata.namespace(conf.namespace) +
+    configMap.mixin.metadata.labels(conf.labels) +
+    configMap.data({"config.json": std.toString(sonobuoyConfigData)});
+
+local pluginConfigs = configMap.new() +
+    configMap.mixin.metadata.name(conf.pluginsCfg.name) +
+    configMap.mixin.metadata.namespace(conf.namespace) +
+    configMap.mixin.metadata.labels(conf.labels) +
+    configMap.data(
+        plugins
+    );
+
+k.core.v1.list.new([
+    sonobuoyConfig,
+    pluginConfigs,
+])
```

Comment:
Tiny, tiny microest of nits, you might consider putting newlines at the end so that diffs are cleaner.